.github/workflows/ci.yaml

```diff
@@ -99,6 +99,11 @@ jobs:
           CUDA_VER: "11.4.2"
           CENTOS_VER: "7"
 
+        - DOCKERIMAGE: linux-anvil-cuda
+          DOCKERTAG: "11.5"
+          CUDA_VER: "11.5.0"
+          CENTOS_VER: "7"
```

Comment:
```suggestion
          CENTOS_VER: "8"
```

Reply:
Why was CentOS 8 needed here as opposed to CentOS 7?

cc @kkraus14